pkg/k8s/diverts/client.go

```diff
@@ -53,6 +54,9 @@ func NewForConfig(cfg *rest.Config) (*DivertV1Client, error) {
 }
 
 func GetClient(thisContext string) (*DivertV1Client, error) {
+	if okteto.IsOktetoContext() {
```

Comment:
I think you don't need this. The divert client should always read from `$HOME/.okteto/contexts/.kubeconfig`